Makefile

```diff
@@ -92,6 +92,35 @@ make test-integration:
 	make -C ./tests/integration test
 
 
+# Changelog management
+## We use Towncrier (https://towncrier.readthedocs.io) for changelog management
+
+ENTRY_TEXT ?= 'Use [Semantic Versioning](https://semver.org/spec/v2.0.0.html) for your changelog entry'
+CHANGE_ID ?= 9999
```

Comment:
There shouldn't be default for `CHANGE_ID`

Reply:
Why not?

Reply:
because its obligatory field and different for every change